### ISSUE TITLE
Shouldn't "power-assert" be in non-dev dependencies?

### ISSUE THREAD

**Issue by wataash** [azu/power-doctest#13]
since I can't execute `power-doctest`:

```
$ npm install -g power-doctest
/usr/local/bin/power-doctest -> /usr/local/lib/node_modules/power-doctest/bin/cmd.js
+ power-doctest@1.3.1
added 121 packages in 5.809s

$ power-doctest a.js
module.js:559
    throw err;
    ^

Error: Cannot find module 'power-assert'
    at Function.Module._resolveFilename (module.js:557:15)
    at Function.Module._load (module.js:484:25)
    at Module.require (module.js:606:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/power-doctest/lib/power-doctest.js:21:20)
    at Module._compile (module.js:662:30)
    at Object.Module._extensions..js (module.js:673:10)
    at Module.load (module.js:575:32)
    at tryModuleLoad (module.js:515:12)
    at Function.Module._load (module.js:507:3)
```

Indeed, `power-assert` is not installed:

```
$ which -a power-doctest
/usr/local/bin/power-doctest

$ ls -lF /usr/local/bin/power-doctest
lrwxr-xr-x 1 wsh admin 44 Feb 27 08:35 /usr/local/bin/power-doctest -> ../lib/node_modules/power-doctest/bin/cmd.js*

$ ls -lF /usr/local/lib/node_modules/power-doctest/bin/cmd.js
-rwxr-xr-x 1 wsh staff 464 Feb 27 08:35 /usr/local/lib/node_modules/power-doctest/bin/cmd.js*

$ ls -F /usr/local/lib/node_modules/power-doctest/node_modules/
acorn/               es6-map/                    optionator/
acorn-es7-plugin/    es6-set/                    os-homedir/
acorn-jsx/           es6-symbol/                 os-tmpdir/
acorn-to-esprima/    es6-weak-map/               path-is-absolute/
ansi-regex/          escallmatch/                power-assert-context-formatter/
ansi-styles/         escape-string-regexp/       power-assert-context-reducer-ast/
array-find/          escodegen/                  power-assert-context-traversal/
ast-source/          escope/                     power-assert-formatter/
babel-code-frame/    espower/                    power-assert-renderer-assertion/
babel-core/          espower-location-detector/  power-assert-renderer-base/
babel-generator/     espree/                     power-assert-renderer-comparison/
babel-helpers/       esprima/                    power-assert-renderer-diagram/
babel-messages/      espurify/                   power-assert-renderer-file/
babel-register/      esrecurse/                  power-assert-util-string-width/
babel-runtime/       estraverse/                 prelude-ls/
babel-template/      esutils/                    private/
babel-traverse/      event-emitter/              process-nextick-args/
babel-types/         fast-levenshtein/           readable-stream/
babylon/             globals/                    regenerator-runtime/
balanced-match/      has-ansi/                   repeating/
brace-expansion/     home-or-tmp/                safe-buffer/
call-matcher/        inherits/                   slash/
chalk/               invariant/                  source-map/
comment-to-assert/   is-finite/                  source-map-support/
concat-map/          is-url/                     string_decoder/
concat-stream/       isarray/                    stringifier/
convert-source-map/  js-tokens/                  strip-ansi/
core-js/             jsesc/                      supports-color/
core-util-is/        json5/                      tagged-template-to-ast/
d/                   levn/                       to-fast-properties/
debug/               lodash/                     traverse/
deep-equal/          loose-envify/               trim-right/
deep-is/             minimatch/                  type-check/
detect-indent/       minimist/                   type-name/
diff-match-patch/    mkdirp/                     typedarray/
eastasianwidth/      ms/                         util-deprecate/
es5-ext/             number-is-nan/              wordwrap/
es6-iterator/        object-assign/              xtend/
```

where all of `dependencies` in `package.json` is installed.
So shouldn't `power-assert` be moved from `devDependencies` to `dependencies`?

I'm using:

- macOS 10.13.3
- npm 5.6.0 (installed via Homebrew)
- node 9.6.1

**Comment by azu** [comment]
Thanks for report.
This module does not depended on `power-assert`.
`power-assert` is run-time library.
Instead of it, this library depended on [espower](https://github.com/power-assert-js/espower "espower").
It just transform code to code.

I've figured out this issue.
This issue come from wrong `build` npm-script.
`build`  script should be run in `NODE_ENV=production`.

https://github.com/azu/power-doctest/blob/cb737b5bf9bd60d053c17e659e92a5cb002ef568/.babelrc#L8
https://github.com/azu/power-doctest/blob/cb737b5bf9bd60d053c17e659e92a5cb002ef568/package.json#L15

**Comment by azu** [comment]
> This module does not depended on power-assert.

But, I think that this library depended on `power-assert` indirectly.
It should be `peerDependencies` or `dependencies`.
I decided that includes `power-assert` as `dependencies`. #15

**Comment by azu** [comment]
I've released [1.3.2](https://github.com/azu/power-doctest/releases/tag/1.3.2 "1.3.2").
@wataash Can you try this?

**Comment by wataash** [comment]
Now it works! thanks :)
